src/seq/nucleotide.jl

```diff
@@ -122,6 +122,16 @@ function isambiguous(nt::Nucleotide)
 end
 
 """
+    isppambiguous(nt::Nucleotide)
+
+Test if nucleotide could be either a Purine or a Pyrimidine.
+"""
+function isppambiguous(nt::Nucleotide)
```

Comment:
Ah this thing again, this is an artefact from before when this PR was distances & also the mutation counting. It got removed in the mutation counting PR as well, I'll remove this.